sdk/purview/purview-workflow-rest/samples-dev/workflowTasksSample.ts

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createPurviewWorkflowClient, {
+  ApproveApprovalTaskParameters,
+  ListWorkflowTasksParameters,
+  paginate,
+  UpdateTaskStatusParameters,
+  RejectApprovalTaskParameters,
+  ReassignWorkflowTaskParameters,
+} from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to Get all workflow tasks.
+ *
+ * @summary Get all workflow tasks.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/ListWorkflowTasks.json
+ */
+
+async function workflowTasksList() {
+  const endpoint = process.env["ENDPOINT"] || "";
+  const tenantId = process.env["TENANTID"] || "";
+  const clientId = process.env["CLIENTID"] || "";
+  const username = process.env["USERNAME"] || "";
+  const password = process.env["PASSWORD"] || "";
+
+  const credential = new UsernamePasswordCredential(tenantId, clientId, username, password);
+  const client = createPurviewWorkflowClient(endpoint, credential);
+  const options: ListWorkflowTasksParameters = {
+    queryParameters: {
+      viewMode: "sent",
+      timeWindow: "30d",
+      maxpagesize: 1000,
+      orderby: "createdTime desc",
+    },
+  };
+  const initialResponse = await client.path("/workflowtasks").get(options);
+  const pageData = paginate(client, initialResponse);
+  const result = [];
+  for await (const item of pageData) {
+    result.push(item);
+  }
+  console.log(result);
+}
+
+/**
+ * This sample demonstrates how to Get a workflow task.
+ *
+ * @summary Get a workflow task.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/GetWorkflowTask.json
+ */
+
+async function workTaskGet() {
+  const endpoint = process.env["ENDPOINT"] || "";
+  const tenantId = process.env["TENANTID"] || "";
+  const clientId = process.env["CLIENTID"] || "";
+  const username = process.env["USERNAME"] || "";
+  const password = process.env["PASSWORD"] || "";
+  const credential = new UsernamePasswordCredential(tenantId, clientId, username, password);
+  const client = createPurviewWorkflowClient(endpoint, credential);
+  const taskId = "98d98e2c-23fa-4157-a3f8-ff8ce5cc095c";
```

Comment:
Please see my previous comment about hard-coded IDs.

Reply:
The  same previous, user could not create task by themselves, so they have to call list workflow tasks api first to get the task id, I provide example and explainations in here: https://github.com/Azure/azure-sdk-for-js/blob/71bdfe63f8576e0b32305f29f970a2e0204a77ad/sdk/purview/purview-workflow-rest/samples/v1-beta/typescript/src/workflowTasksSample.ts#L178